### DISCUSSION TITLE
VGA sprites

### DISCUSSION THREAD

**Discussion by brentward** [picocomputer/community#36] [Q&A]
I want to make a sprite POC. I was looking at the VGA documentation for the "new design" that exists in the vga branch and given the vga branch was merged into main for release 0.1 I was wondering if it is possible.

After looking through the code it seems to not be currently implemented, but I just wanted to check that I'm not missing anything.

I did my best to put the proper structures in xram and I tried to set the correct PIX vga registers using xreg but I didn' tget a sprite on screen.
This is the code I used to draw a sprite:
https://github.com/brentward/hello_vga

**Comment by brentward** [comment]
I just saw @rumbledethumps new video. It seemed to answer the question that sprites indeed aren't implemented yet in 0.1. I'm looking forward to trying out some more complex graphics when it is ready though.

**Reply by michaelsarr** [reply]
yeah, it will be pretty cool when it gets here
